schemata/pdf_manager_zip_file.json

```diff
@@ -1,168 +0,0 @@
-{
```

Comment:
Thibault and I talked about this.  This is a leftover from a previous version of the PDF Manager API.